src/scratchtocatrobat/scratch/scratch3visitor/test_scratch3.py

```diff
@@ -1326,6 +1326,11 @@ def test_visitLength(self):
         assert converted_block[0] == "stringLength:"
         assert converted_block[1] == "teststring"
 
+    def test_visit_literal(self):
```

Comment:
In case of this test it might also be a good idea to test the other possible values of literal[0] with one example of correct input. Eg. if literal[0] == 12 check for "readVariable" and literal[1]. Those checks could be included in this test directly.